frontend/src/contracts.ts

```diff
@@ -351,7 +331,7 @@ export async function setUpContracts(web3: Web3): Promise<Contracts> {
     const simpleQuests = new web3.eth.Contract(simpleQuestsAbi as Abi, simpleQuestsContractAddr);
     questsContracts.SimpleQuests = simpleQuests;
     if (simpleQuests.methods.partnerVault) {
```

Comment:
This if is checking if partnerVault property is in SimpleQuests contract in order to get the address from there. Now it's not needed I think

Reply:
removed